src/coverlet.console/coverlet.console.csproj

```diff
@@ -24,6 +24,7 @@
   </PropertyGroup>
  
   <ItemGroup>
+    <PackageReference Include="NuGet.Versioning" />
```

Comment:
Why this needed here? Don't we already use `Nerdbank.GitVersioning` as a global refernce in all packable projects? The only real reference to `NuGet.Versioning` seems to be in `coverlet.core`. But maybe I'm missing something.